addons/website_timesheet/models/account_analytic_line.py

```diff
@@ -0,0 +1,13 @@
+from odoo import api, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    @api.model
+    def _show_portal_timesheets(self):
+        """
+        Determine if we show timesheet information in the portal.
+        """
+        domain = [("key", "=", "hr_timesheet.portal_my_home_timesheet")]
+        return self.env["ir.ui.view"].sudo().with_context(active_test=False).search(domain).filter_duplicate().active
```

Comment:
Does it not equivalent to: 

```suggestion
        return len(self.env["ir.ui.view"].sudo().search(domain).filter_duplicate()) >= 1
```

Reply:
Apparently not :thinking: Your suggested solution does not work

Reply:
:o